src/pages/MyTokens/NFTPage.tsx

```diff
@@ -106,7 +106,7 @@ export const NFTPage = () => {
     setIsFetchingTokens(true);
     void (async () => {
       await fetch({ page: 1, pageSize, seller: selectedAccount?.address });
-      const _tokens = await api.nft?.getAccountMarketableTokens(selectedAccount.address) as NFTToken[];
```

Comment:
:warning: Just to simplify refactoring for self - remove "NFTToken" from code and follow errors.